pyproject.toml

```diff
@@ -37,7 +35,7 @@ pysolr = "^3.9.0"
 eutils = ">=0.6.0"
 requests-cache = "^1.0.1"
 click = "*"
-semsimian = "0.2.1"
+semsimian = ">=0.2.1"
```

Comment:
In general I favor less pinning but given how tightly coupled these are, I think limiting to `0.2.*` may be sensible? See also Mark's Q in #oak

Reply:
ok, switched to `^0.2.1`. As per Mark's Q, I did read it and it was confusing with the versions he mentioned.